### ISSUE TITLE
unable to find version `v1`

### ISSUE THREAD

**Issue by nyonson** [ScaCap/code-coverage-assistant#43]
My old v0.1 workflow stopped working, maybe due to the version bump? Tried changing to v1 like in the readme, but getting the following:

```
Error: Unable to resolve action `ScaCap/code-coverage-assistant@v1`, unable to find version `v1`
```

**Comment by Ewocker** [comment]
the version has been off since I start using it and I can never really figure out the right version vs repo...

**Comment by nyonson** [comment]
Looks like `v1.0` works

**Comment by eeshdarthvader** [comment]
@nyonson I have published the `v1.0` [last week](https://github.com/ScaCap/code-coverage-assistant/releases/tag/v1.0). Sorry for the confusion, now it should all work.
